log/zap/zap_sugar_logger_test.go

```diff
@@ -0,0 +1,116 @@
+package zap_test
+
+import (
+	"encoding/json"
+	kitzap "github.com/go-kit/kit/log/zap"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+	"strings"
+	"testing"
+)
+
+type tbWriter struct {
+	tb testing.TB
+	sb strings.Builder
+}
+
+func (w *tbWriter) Write(b []byte) (n int, err error) {
+	w.tb.Logf(string(b))
+	w.sb.Write(b)
+	return len(b), nil
+}
+
+var encoderConfig = zap.NewDevelopmentEncoderConfig()
+
+func newLogger(writer zapcore.WriteSyncer, enabler zapcore.LevelEnabler) *zap.Logger {
+	return zap.New(
+		zapcore.NewCore(zapcore.NewJSONEncoder(encoderConfig), writer, enabler),
+		zap.Development())
+}
+
+type testCase struct {
+	level zapcore.Level
+	kvs   []interface{}
+	want  map[string]string
+}
+
+var testCases = []testCase{
+	{
+		zapcore.DebugLevel,
+		[]interface{}{"key1", "value1"},
+		map[string]string{
+			encoderConfig.LevelKey: "DEBUG",
+			"key1":                 "value1",
+		}},
+	{
+		zapcore.InfoLevel,
+		[]interface{}{"key2", "value2"},
+		map[string]string{
+			encoderConfig.LevelKey: "INFO",
+			"key2":                 "value2",
+		}},
+	{
+		zapcore.WarnLevel,
+		[]interface{}{"key3", "value3"},
+		map[string]string{
+			encoderConfig.LevelKey: "WARN",
+			"key3":                 "value3",
+		}},
+	{
+		zapcore.ErrorLevel,
+		[]interface{}{"key4", "value4"},
+		map[string]string{
+			encoderConfig.LevelKey: "ERROR",
+			"key4":                 "value4",
+		}},
+	{
+		zapcore.DPanicLevel,
+		[]interface{}{"key5", "value5"},
+		map[string]string{
+			encoderConfig.LevelKey: "DPANIC",
+			"key5":                 "value5",
+		}},
```

Comment:
Please split the closing braces onto their own line so that they balance the opening braces at the same indentation level.